src/PKSim.CLI.Core/Services/QualificationRunner.cs

```diff
@@ -125,10 +124,5 @@ private Simulation simulationFrom(Project snapshotProject, string simulationName
          var referenceSimulation = snapshotProject.Simulations?.FindByName(simulationName);
          return referenceSimulation ?? throw new QualificationRunException(CannotFindSimulationInSnapshot(simulationName, snapshotProject.Name));
       }
-
-      protected override Task ExportToMarkdown(object buildingBlock, string fileFullPath, int? inputSectionLevel)
```

Comment:
before this naming made sense. We are exporting to a file and return void (Task). No surprise here

Reply:
We are still going to export the markdown in the `LoadProjectAndExportInputs`. Do you mean we should rename it `LoadProjectAndExportMarkdown` instead?

Reply:
I guess what's unexpected is "export" that does not take a file name but returns something and in fact it does export to file and still returns. This is not a typical signature that's why it tripped me